### PR TITLE
Implemented the listing of financial lists for a display

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

**Comment by settinghead** [comment]
@tejohnso please review. Thanks. @Rise-Vision/delivery FYI

File: src/components/displays/displays.html [hunk 1]
```diff
@@ -63,7 +64,7 @@
 
       <tbody class="table-body">
 
-        <tr class="table-body__row" ng-repeat="display in $ctrl.displayList">
+        <tr e2e-key="{{display.$id}}" class="table-body__row" ng-repeat="display in $ctrl.displayList">
```
Comment (tejohnso) [inline src/components/displays/displays.html]: @settinghead Can we avoid putting an attribute here just for e2e tests and instead use the `{{display.displayName}}` in the anchor element below?

**Comment by settinghead** [comment]
@tejohnso fixed re. getting rid of extra e2e attributes. Thanks.

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**